crates/client/sync/src/metrics/block_metrics.rs

```diff
@@ -7,6 +7,7 @@ pub struct BlockMetrics {
     pub l2_sync_time: Gauge<F64>,
     pub l2_avg_sync_time: Gauge<F64>,
     pub l2_latest_sync_time: Gauge<F64>,
+    pub l2_state_size: Gauge<F64>,
```

Comment:
I'd like a HistogramVec for every column if possible :)

Reply:
doing it rn